src/main/java/com/iota/iri/TransactionValidator.java

```diff
@@ -266,6 +268,11 @@ private boolean checkApproovee(TransactionViewModel approovee) throws Exception
         return approovee.isSolid();
     }
 
+    //for testing
+    boolean isNewSolidTxSetsEmpty () {
```

Comment:
method should have explicit visibility? (or did we consciously make the choice to allow this?)

Reply:
ditto